config/routes.rb

```diff
@@ -4,9 +4,17 @@
   # Defines the root path route ("/")
   # root "articles#index"
   # gets 'authors/:id', to: "authors#show"
-  # resources :authors, :posts
-  get "/authors/:id/posts", to: "posts#index"
+  resources :authors do
+    resources :posts do
+      resources :comments
+    end
+  end
+  
+  get "/authors/:id/posts/new", to: "posts#new", as: 'new_author_post_path'
+  get "/authors/:author_id/posts/:post_id/like", to: "likes#create"
   get "/authors/:author_id/posts/:id", to: "posts#show"
+  post "/authors/:author_id/posts/:post_id/", to: "posts#create", as: 'post'
+  post "/authors/:author_id/posts/:post_id/comments", to: "comments#create", as: 'comment'
   get "/authors", to: "authors#index"
   get "/authors/:id", to: "authors#show"
```

Comment:
- [ ] [OPTIONAL] Your routes currently look like this: `/authors/:id`. A more ideal route if you had followed the [ERD diagram](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_app_erd.png) would have been `/users/:id` because in real life, not all users are authors, but all authors are users of the application / blog.
In order to rectify this, the model, table, controller and routes need to be appropriately named and set up👍